contracts/base/Escrow.sol

```diff
@@ -153,7 +153,10 @@ contract Escrow is EscrowInterface, TInitializable, BaseEscrowDapp {
         @dev The recipient must be the loan borrower AND the loan must be already liquidated.
     */
     function claimTokens() external onlyOwner() {
-        require(getLoan().status == TellerCommon.LoanStatus.Closed, "LOAN_NOT_CLOSED");
+        require(
+            getLoan().status == TellerCommon.LoanStatus.Liquidated,
```

Comment:
Because this function can only be called by the owner, we should not be checking that the loan is `Liquidated` but rather just `Closed`. If a user pays back their loan, we still want them to be able to claim their tokens that are still in the Escrow.